<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gb_dcc</class>
 <widget class="QMainWindow" name="gb_dcc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>343</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gertbot DCC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gb_dcc.qrc">
    <normaloff>:/gb_dcc/Resources/train.bmp</normaloff>:/gb_dcc/Resources/train.bmp</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>19</width>
    <height>29</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="pb_pointA">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 08/09</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/point2.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/point1.bmp</normalon>:/gb_dcc/Resources/point2.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_signalA">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 18/19</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/sign_green.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/sign_red.bmp</normalon>:/gb_dcc/Resources/sign_green.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>29</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pointA">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pnt A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pointB">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pnt B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_pointB">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 0A/0B</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/point2.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/point1.bmp</normalon>:/gb_dcc/Resources/point2.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sigA">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sig A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_signalB">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 1A/1B</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/sign_green.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/sign_red.bmp</normalon>:/gb_dcc/Resources/sign_green.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>29</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sigB">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sig B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_pointD">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 0E/0F</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/point2.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/point1.bmp</normalon>:/gb_dcc/Resources/point2.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pointC">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>140</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pnt C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_pointC">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 0C/0D</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/point2.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/point1.bmp</normalon>:/gb_dcc/Resources/point2.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pointD">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>190</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pnt D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sigA_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sig C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_signalC">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 1C/1D</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/sign_green.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/sign_red.bmp</normalon>:/gb_dcc/Resources/sign_green.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>29</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="pb_signalD">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Adrs=80 1E/1F</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="gb_dcc.qrc">
      <normaloff>:/gb_dcc/Resources/sign_green.bmp</normaloff>
      <normalon>:/gb_dcc/Resources/sign_red.bmp</normalon>:/gb_dcc/Resources/sign_green.bmp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>29</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sigB_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sig D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>184</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="spB_loco1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="sld_loc1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="pb_Stop_loc1">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="gb_dcc.qrc">
             <normaloff>:/gb_dcc/Resources/stop.bmp</normaloff>:/gb_dcc/Resources/stop.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QSpinBox" name="spB_loco2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="sld_loc2">
           <property name="toolTip">
            <string>Loc=02</string>
           </property>
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="pb_Stop_loc2">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/gb_dcc/Resources/stop.bmp</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QSpinBox" name="spB_loco3">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="sld_loc3">
           <property name="toolTip">
            <string>Loc=08</string>
           </property>
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="pb_Stop_loc3">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/gb_dcc/Resources/stop.bmp</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QToolButton" name="pb_stop_all">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>106</red>
             <green>104</green>
             <blue>100</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop All Trains</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="te_log">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="gb_dcc.qrc"/>
 </resources>
 <connections/>
</ui>
